.nuke/build.schema.json

```diff
@@ -37,6 +37,7 @@
         "RunLocalClient",
         "RunLocalServer",
         "TestServer",
+        "TestServerE2e",
```

Comment:
Sorry, my bad, I should have specified the Nuke target name as TestE2e, can you change it to that?